airflow/www/static/js/components/Clipboard.tsx

```diff
@@ -43,9 +43,13 @@ export const ClipboardButton = forwardRef(
     },
     ref
   ) => {
-    const { hasCopied, onCopy } = useClipboard(value);
+    const { setValue, hasCopied, onCopy } = useClipboard(value);
     const containerRef = useContainerRef();
 
+    useEffect(() => {
+      setValue(value);
```

Comment:
Can we wrap this in an if statement and only setValue if it is different from the current clipboard value?

Reply:
After reading https://stackoverflow.com/questions/41322523/reactjs-calling-setstate-with-same-parameter carefully, I believe this is not necessary, as same values do not trigger a state change event.

Would this unblock the merge request - allowing to close #34846 ?

Reply:
WDYT @bbovenzi ?